app/src/ereferrals/java/org/eyeseetea/malariacare/data/sync/importer/metadata/configuration/IMetadataConfigurationDataSource.java

```diff
@@ -0,0 +1,11 @@
+package org.eyeseetea.malariacare.data.sync.importer.metadata.configuration;
+
+
+import org.eyeseetea.malariacare.domain.entity.Question;
+
+import java.util.List;
+
+public interface IMetadataConfigurationDataSource {
```

Comment:
Ok, I agree with moving the interface to main, but I am a little confuse with putting it on **data.remote** package because there's also a **data.sync.import** package where there's similar information.

For example the classes related to CVSImporter are in the same package **data.sync.import** but in the **ereferals** variant.

Could you help, please, to clarify in  which one should I put the interface?